design/ocwg-design-decisions.md

```diff
@@ -180,10 +131,9 @@ minimal structure for resources:
     @enduml
     -->
 
-## Content-Formats
-
-Canvas tools often support some special content formats, especially
-free-hand drawing. We might want to define a content for this, so that
+### Content-Formats
+Canvas tools often support some special content formats, e.g., 
+free-hand drawings. We might want to define a content for this, so that
 it can be exchanged.
 
 Idea: Represent the actual path useing the SVG
```

Comment:
Should probably cut this 
```suggestion
```

Cut this? It's mentioned elsewhere.

Reply:
Well... as far as I remember, we had the idea to later mint some custom MIME types for e.g. free-hand drawings. Theses are frequently used in canvas apps, but no obvious MIME type exists. SVG path data seems to be the closest existing standard.